dev/flutter/todo/submissions/Dishal Kanishka/todo_application/lib/Class/theem.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+import 'package:todo_application/Constants/colors.dart';
+
+class ThemeClass {
+  static ThemeData lightTheme = ThemeData(
+      primaryColor: Colors.black,
+      primaryColorLight: Colors.black,
+      brightness: Brightness.light,
+      scaffoldBackgroundColor: Colors.white,
+      colorScheme: ColorScheme.light(),
+      appBarTheme: AppBarTheme(
+        backgroundColor: kAppBarColor,
+      ),
+      canvasColor: Colors.white,
+      bottomAppBarTheme: BottomAppBarTheme(color: kAppBarColor));
+
+  static ThemeData darkTheme = ThemeData(
+      primaryColor: Color(0xFF80CBC4),
+      primaryColorLight: Colors.white,
+      scaffoldBackgroundColor: Colors.black,
+      brightness: Brightness.dark,
+      colorScheme: ColorScheme.dark(),
+      appBarTheme: AppBarTheme(
+        backgroundColor: Colors.black,
+      ),
+      canvasColor: Colors.black,
+      bottomAppBarTheme: BottomAppBarTheme(color: Colors.black));
+}
```

Comment:
The filename of this has a typo, and let's follow a single naming convention. I see that this folder is in upper camel case and the screens folder is lowercase. 

Also it's more appropriate to put it within a folder such as `config` rather than just `Class`. Class is too generic. Same goes for the todo. It's better to put it within a `models` or `entities` folder so that it has meaning

Reply:
Correct typing mistake and config folder created for Class and models folder created and todo.dart put to it